src/screens/regionPicker/RegionPicker.tsx

```diff
@@ -0,0 +1,129 @@
+import React, {useState} from 'react';
+import {useNavigation} from '@react-navigation/native';
+import {Box, Text, Icon, Button} from 'components';
+import {StyleSheet, TouchableOpacity, ScrollView, Image} from 'react-native';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {useStorage} from 'services/StorageService';
+import {useI18n} from '@shopify/react-i18n';
+import AbFlag from 'assets/flags/ab-flag.png';
+import BcFlag from 'assets/flags/bc-flag.png';
+import MbFlag from 'assets/flags/mb-flag.png';
+import NbFlag from 'assets/flags/nb-flag.png';
+import NlFlag from 'assets/flags/nl-flag.png';
+import NtFlag from 'assets/flags/nt-flag.png';
+import NsFlag from 'assets/flags/ns-flag.png';
+import NuFlag from 'assets/flags/nu-flag.png';
+import OnFlag from 'assets/flags/on-flag.png';
+import PeFlag from 'assets/flags/pe-flag.png';
+import QcFlag from 'assets/flags/qc-flag.png';
+import SkFlag from 'assets/flags/sk-flag.png';
+import YtFlag from 'assets/flags/yt-flag.png';
+import {Region} from 'shared/Region';
+
+interface RegionItemProps {
+  code: Region;
+  flagIcon: any;
+  name: string;
+  selected: boolean;
+  onPress: (code: Region) => void;
+}
+
+const items: Omit<RegionItemProps, 'onPress' | 'selected' | 'name'>[] = [
+  {code: 'AB', flagIcon: AbFlag},
+  {code: 'BC', flagIcon: BcFlag},
+  {code: 'MB', flagIcon: MbFlag},
+  {code: 'NB', flagIcon: NbFlag},
+  {code: 'NL', flagIcon: NlFlag},
+  {code: 'NT', flagIcon: NtFlag},
+  {code: 'NS', flagIcon: NsFlag},
+  {code: 'NU', flagIcon: NuFlag},
+  {code: 'ON', flagIcon: OnFlag},
+  {code: 'PE', flagIcon: PeFlag},
+  {code: 'QC', flagIcon: QcFlag},
+  {code: 'SK', flagIcon: SkFlag},
+  {code: 'YT', flagIcon: YtFlag},
+];
+
+const RegionItem_ = ({code, onPress, name, flagIcon, selected}: RegionItemProps) => (
+  <>
+    <TouchableOpacity onPress={() => onPress(code)}>
```

Comment:
I think we need accessibilityRole = "radio" on these, similar to https://github.com/CovidShield/mobile/issues/65